ui/app/templates/components/auth-form.hbs

```diff
@@ -32,18 +32,16 @@
               </li>
             {{/let}}
           {{/each}}
-          {{#if this.hasMethodsWithPath}}
```

Comment:
This condition is already wrapped higher up correct? It looks like removing it is the only change here.

Reply:
Yeah - meant to comment. It looked like this was redundant? So removed to clean it up - but want to make sure I'm not misreading...